packages/storybook-react/src/stories/template/PageStep-2.stories.tsx

```diff
@@ -20,101 +25,241 @@ import {
+      validateField(dateRef.current);
+      validateField(postcodeRef.current);
+      validateField(homeNumberRef.current);
+    };
+
+    return (
+      <Page {...args} style={pageStyle}>
+        <PageHeader>
+          <Link href="/" aria-label="Gemeente Utrecht">
+            <UtrechtLogo />
+          </Link>
+        </PageHeader>
+        <PageContent role="main">
+          <BreadcrumbNav appearance="arrows" style={{ marginBlockEnd: '1.5rem' }}>
+            <BreadcrumbLink href="https://example/" index={0} current={false} rel="home">
+              Home
+            </BreadcrumbLink>
+            <BreadcrumbLink href="https://example/" index={0} current={true} rel="Online loket">
+              Online loket
+            </BreadcrumbLink>
+          </BreadcrumbNav>
+          <HeadingGroup>
+            <Heading1>Een verhuizing doorgeven</Heading1>
+            <Paragraph>Stap 2 van 5 — De verhuizing</Paragraph>
+          </HeadingGroup>
+          <div style={rowStyle}>
+            <div style={descriptionStyle}>
+              <Heading2>De verhuizing</Heading2>
+              <Separator />
             </div>
-            <FormField>
-              <FormLabel htmlFor="postcode">Postcode *</FormLabel>
-              <br />
-              <Textbox id="postcode" required aria-invalid="true" aria-errormessage="err-postcode" />
-              <UtrechtFormFieldDescription id="err-postcode" status="invalid">
-                U moet de postcode van het nieuwe adres invullen.
-              </UtrechtFormFieldDescription>
-            </FormField>
-            <FormField>
-              <FormLabel htmlFor="huisnummer">Huisnummer *</FormLabel>
-              <br />
-              <Textbox id="huisnummer" required />
-            </FormField>
-            <UtrechtFormFieldDescription>Vragen met een * zijn verplicht.</UtrechtFormFieldDescription>
-          </Fieldset>
-          <Link href="/">VORIGE</Link>
-          <Button type="submit" appearance="primary-action-button">
-            VOLGENDE
-          </Button>
-        </form>
-      </section>
-    </PageContent>
-    <PageFooter />
-  </Page>
-);
+            <aside style={logoButtonGroupStyle}>
+              <Heading3>MIJN LOKET</Heading3>
+              <UtrechtDigidButton>
+                <ButtonLink appearance="primary-action-button">
+                  Uitloggen
+                  <UtrechtIconArrow />
+                </ButtonLink>
+              </UtrechtDigidButton>
+              <Paragraph>Mevrouw Cindy</Paragraph>
+            </aside>
+          </div>
+          <section>
+            <form style={{ width: '70%' }} action="/api/adres" method="POST" onSubmit={onNewAddressSubmit} noValidate>
+              <FormField invalid={invalid}>
+                <p>
+                  <FormLabel htmlFor="verhuisdatum">Verhuisdatum*</FormLabel>
+                </p>
+                <Textbox
+                  onChange={onNewAddressChange}
+                  aria-required
```

Comment:
Zouden we niet `required` moeten gebruiken in plaats van `aria-required`?

Reply:
Als ik de `required`  attribute gebruik, krijg ik de input met rode border als invalid, 
Maar ik ga het veranderen tot dat wij een oplossen kunnen vinden